client/src/renderer/redux/actions.js

```diff
@@ -18,3 +22,9 @@ export const updateCaretPos = ({ positionStart, positionEnd }) => ({
     positionEnd,
   },
 });
+
+export const setMetaData = (key, data) => ({
```

Comment:
For consistency, please treat metadata as one word.

```suggestion
export const setMetadata = (key, data) => ({
```